dist/spec.json

```diff
@@ -13,7 +13,7 @@
       "name": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "2.5.0"
+    "version": "2.6.1"
```

Comment:
should it be 2.6.0?

Reply:
Updated due to comments above, but in case 2.6.0 hasn't been released yet, I agree, we can use 2.6.0 here